src/common/mfu_util.c

```diff
@@ -195,6 +197,45 @@ int daos_connect(
 
     return 0;
 }
+
+int daos_mount(
+  mfu_file_t* mfu_file,
+  daos_handle_t* poh,
+  daos_handle_t* coh)
+{
+    int rc = 0;
+
+    /* Mount dfs */
+    rc = dfs_mount(*poh, *coh, O_RDWR, &mfu_file->dfs);
+    if (rc != 0) {
+        MFU_LOG(MFU_LOG_ERR, "Failed to mount DAOS filesystem (DFS): "
+                MFU_ERRF, MFU_ERRP(-MFU_ERR_DAOS));
+        rc = -1;
+    }
+
+    return rc;
+}
+
+int daos_umount(
+  mfu_file_t* mfu_file)
+{
+    int rc = 0;
+    int tmp_rc;
+    
+    /* Unmount dfs */
+    tmp_rc = dfs_umount(mfu_file->dfs);
```

Comment:
Just a style thing here in trying to declare locals where they are first used.
```suggestion
    
    /* Unmount dfs */
    int tmp_rc = dfs_umount(mfu_file->dfs);
```

Reply:
Ah, yes. It has been hammered in my brain to do it the other way. But I actually do prefer this way.